responsibleai/requirements.txt

```diff
@@ -7,6 +7,7 @@ lightgbm>=2.0.11
 numpy>=1.17.2
 numba<0.54.0
 pandas>=0.25.1
+raimitigations~=0.0.3
```

Comment:
I see this on PyPI https://pypi.org/project/raimitigations/
Will it be public soon?

Reply:
Yes, it's currently hosted in a private Github repo called "responsible-ai-mitigations" and both the repo and package will be public soon!

Reply:
It would be nice to see that publicly on GitHub first, because otherwise we're integrating something private into this open source tool which isn't quite in the spirit of open source 🙂